app/views/facilities/_product_list.html.haml

```diff
@@ -1,18 +1,22 @@
 - if products.any?
   .product_list{ class: products.first.class.model_name.to_s.pluralize.downcase }
-    %h3= product_list_title products, local_assigns[:title_extra]
+
+    - if current_facility.present?
+      %h3= product_list_title products, local_assigns[:title_extra]
+
     - if products.first.class.model_name == "TimedService"
```

Comment:
I think the TimedService note ought to be nested under the `if current_facility.present?` as well. Try adding a Timed Service product type to see how it shows up.